packages/manager/apps/web/client/app/hosting/database/DATABASE_LIST.html

```diff
@@ -113,7 +113,14 @@
                         <tr>
                             <th scope="col" data-translate="hosting_tab_DATABASES_table_header_name"></th>
                             <th scope="col" data-translate="hosting_tab_DATABASES_table_header_database_name"></th>
-                            <th scope="col" data-translate="hosting_tab_DATABASES_table_header_server_address"></th>
+                            <th scope="col">
+                                <span id="serverAddressHeader" data-translate="hosting_tab_DATABASES_table_header_server_address"></span>
+                                <span class="fa fa-info-circle"
+                                      aria-hidden="true"
+                                      data-uib-tooltip="{{ 'hosting_tab_DATABASES_table_header_server_address_tooltip' | translate }}"
```

Comment:
Please use a `oui-tooltip` instead :)

Reply:
It looks like the oui-popover doesn't work that well in a table.

Reply:
Did you write this piece of code:

```js
<button
    type="button"
    class="oui-popover-button"
    oui-popover="{{:: 'hosting_tab_DATABASES_table_header_server_address_tooltip' | translate }}"></button>
```

Reply:
Yes I did, but the popover is not located at the correct place. I think this is because we are in a table.
![popover](https://user-images.githubusercontent.com/5231539/70984999-b7a15c00-20bb-11ea-8113-32ac42d9ea1e.png)